src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,165 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.InvalidDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceTest {
+    private static final int ADULT_AGE = 18;
+    private static final int ANOTHER_ADULT_AGE = 25;
+    private static final int NEGATIVE_AGE = -10;
+    private static final int TEEN_AGE = 17;
+    private static final int ZERO_AGE = 0;
```

Comment:
You have hardcoded values for ages, which may not be relevant for the task requirements. Consider removing them if they are not used to handle cases that are likely to change over time, such as the maximum human lifespan.